.github/scripts/end2end/run-e2e-pra-ctst.sh

```diff
@@ -0,0 +1,144 @@
+#!/bin/bash
+set -exu
+
+ZENKO_NAME=${1:-end2end}
+COMMAND=${2:-"premerge"}
+PARALLEL_RUNS=${PARALLEL_RUNS:-$(( ( $(nproc) + 1 ) / 2 ))}
+RETRIES=${4:-3}
+JUNIT_REPORT_PATH=${JUNIT_REPORT_PATH:-"ctst-junit.xml"}
```

Comment:
may need different file names for PRA and the other tests, not sure we will "merge" artifacts properly otherwise...